release/builder/build.sh

```diff
@@ -24,16 +24,22 @@ apt-get install gnupg2 -y
 # Install Java
 apt-get install openjdk-11-jdk-headless -y
 # Install Python
-apt-get install python -y
+echo ">>>>"
+apt-get install python
+echo ">>>> 2"
+apt-get install python3.8
```

Comment:
I don't think there is a python3.8 package in the repo, at least there's not in any of the ones I'm using -- `apt-get install python3 -y` is what you want.  In fact, there's no "python" package any more, the old version has become "python2".  This might be the reason for the problem you're seeing.

You need the -y to suppress the "are you sure you want to install..." prompt.